support/sys-xattr.c

```diff
@@ -12,7 +12,7 @@
 #include <config.h>
 
 //If we're compiling to API level < 16 this won't be available
-#if __ANDROID_API__ < 16
+#if defined (HOST_ANDROID) && __ANDROID_API__ < 16
```

Comment:
Why does this #if succeed on non-android platforms ?

Reply:
undefined evaluates as 0, silently, per the language. Microsoft has a warning for this but it isn't enabled by default and really it is just expected behavior and a warning is dubious

I suggest similar:

jays-MacBook-Pro:support jay$ git diff -u
+#ifdef __ANDROID_API__
 #if __ANDROID_API__ < 16
 #define ANDROID_NO_XATTR
 #endif
+#endif